virttest/qemu_vm.py

```diff
@@ -1685,27 +1686,27 @@ def __iothread_conflict_check(params):
             :type params: dict
             """
             iothread_scheme = params.get("iothread_scheme")
-            image_iothread_vq_mapping = params.get(
-                "image_iothread_vq_mapping")
+            image_iothread_vq_mapping = params.get("image_iothread_vq_mapping")
             iothreads_lst = params.objects("iothreads")
             # The legacy 'iothread_scheme' does NOT support the
             # parameter 'iothread_vqs_mapping'. If you're going to use the legacy
             # 'iothread_scheme', the 'iothread_vqs_mapping' must NOT be set.
             if (iothread_scheme and image_iothread_vq_mapping) or (
-                    iothread_scheme in ("multipeerroundrobin", "full"
-                                        ) and iothreads_lst):
```

Comment:
Same as the above.

Reply:
> Same as the above.

If I do NOT change the style, the CI static check gives me a fail result.

Reply:
> > Same as the above.
> 
> If I do NOT change the style, the CI static check gives me a fail result.

Yeah, I understand your concern, maybe we move those changes to the commit: https://github.com/avocado-framework/avocado-vt/pull/3785/commits/b929d47957776e5b018bf9891a070764f59d920f, not here.